Source/WebCore/platform/mac/ThemeMac.mm

```diff
@@ -421,6 +433,10 @@ static LengthSize switchSize(const LengthSize& zoomedSize, float zoomFactor)
     case StyleAppearance::InnerSpinButton: {
         static const int stepperMargin[4] = { 0, 0, 0, 0 };
```

Comment:
I think you also need to add `stepperMargins` to match the cell outsets. Otherwise, I would expect issues (painting artifacts) on repaint.